auth/src/main/java/org/jboss/pnc/auth/JAASLoggedInUser.java

```diff
@@ -39,7 +39,7 @@ public JAASLoggedInUser(HttpServletRequest httpServletRequest) {
 
     @Override
     public String getEmail() {
-        return "Email N/A (" + getUserName() + ")";
+        return "example@example.com";
```

Comment:
User entity inserted only when the user does not exists. We have "auto-registration" for authenticated users. You can insert a user with proper email and matching username, before the tests.